firmware/Core/Inc/config/configuration.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __INCLUDE_GUARD__CONFIGURATION_H
+#define __INCLUDE_GUARD__CONFIGURATION_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef struct
+{
+    const char *variable_name;
+    uint32_t *num;
```

Comment:
`num` is a weak variable name. Perhaps `pointer_to_config_var`? Or just `config_var`?

Reply:
renamed to num_config_var